flink-connector-redis/src/main/java/org/apache/flink/streaming/connectors/redis/RedisTableSinkFactory.java

```diff
@@ -57,6 +57,7 @@
         properties.add(REDIS_COMMAND);
         properties.add(REDIS_NODES);
         properties.add(REDIS_MASTER_NAME);
+        properties.add(CLUSTER_PASSWORD);
```

Comment:
Could we follow the same pattern for the variables (e.g. `REDIS_CLUSTER_PASSWORD`)